test/helpers/client.js

```diff
@@ -41,6 +44,24 @@ class Client {
         this._respondWithCallback(JSON.parse(data));
     }
 
+    _checkError(error) {
+        // Assumes that an instance of the server is not running, so lets start one.
+        if (error.errno === "ECONNREFUSED") {
+            if (!this.child) {
+                this.child = fork('main.js', ['grunt', '--debug', 'silly'])
+            }
+            setTimeout(() => {
+                this.ws = new WebSocket(`ws://localhost:${this.port}`);
+                this.ws.on('open', this._onOpen.bind(this));
+                this.ws.on('close', this._onClose.bind(this));
+                this.ws.on('message', this._onMessage.bind(this));
+            }, 5000);
```

Comment:
I increased the timeouts in the past because on travis it took up to 20sec for the websocket server to be created (SE Linux...)